### PR TITLE
New package: materia-theme-20191017

### PR DESCRIPTION
**Pull request by toluschr**

### CODE REVIEW

**Comment by ndowens** [comment]
Get rid of INSTALL.msg ; tends to be only accepted if it is important, instead of information

File: srcpkgs/materia-theme/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+maintainer="toluschr <toluschr@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/nana-4/${pkgname}"
+distfiles="https://github.com/nana-4/${pkgname}/archive/v${version}.tar.gz"
+checksum=ab60f2b79f4b22cefa432d00bae8456e27073d2ba22297da0ea92ac09c994d71
+
+do_install() {
+	./install.sh -d $DESTDIR/usr/share/themes
+}
+
```
Comment (ndowens) [inline srcpkgs/materia-theme/template]: Remove empty last line.

**Comment by toluschr** [comment]
I made the changes and squashed the commits.

**Comment by sgn** [comment]
Committer said no to this package, see https://github.com/void-linux/void-packages/pull/14247

**Pull request closed without merging**